scripts/ci/build_environment.sh

```diff
@@ -0,0 +1,6 @@
+conda install -c conda-forge mamba
```

Comment:
Do we want to use mamba to build the env?

Reply:
Yes, good catch!